protocol/contracts/templegold/EpochLib.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.8.20;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Temple (templegold/EpochLib.sol)
+
+import { IAuctionBase } from "contracts/interfaces/templegold/IAuctionBase.sol";
+
+library EpochLib {
+
+    function isActive(IAuctionBase.EpochInfo storage info) internal view returns (bool) {
+        return info.startTime <= block.timestamp && block.timestamp < info.endTime;
+    }
+
+    function hasEnded(IAuctionBase.EpochInfo storage info) internal view returns (bool) {
+        return info.endTime <= block.timestamp;
+    }
+
+    function hasStarted(IAuctionBase.EpochInfo storage info) internal view returns (bool) {
```

Comment:
This function is not used anywhere (and it is a pitty because it would have made clearer some of the requirements in the auction contracts)

Reply:
Got it. It used to be, but after some first reviews , it was replaced with `hasEnded()` in some corner cases